content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -98,7 +98,7 @@ update-alternatives --set iptables /usr/sbin/iptables-legacy
 | TCP      | Inbound   | 10250       | Kubelet API           | Self, Control plane     |
 | TCP      | Inbound   | 30000-32767 | NodePort Services**   | All                     |
 
-** Default port range for [NodePort Services](/docs/concepts/services-networking/service/).
```

Comment:
The original line matches `NodePort Services**`.

Maybe change from `**` to `†`?